plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/server/debug/OpenShiftDebugUtils.java

```diff
@@ -298,21 +298,20 @@ public void setupRemoteDebuggerLaunchConfiguration(ILaunchConfigurationWorkingCo
 		}
 	}
 	
-	public boolean isRunning(ILaunchConfiguration launchConfiguration, int localDebugPort) {
+	public boolean isRunning(ILaunchConfiguration launchConfiguration) {
 		boolean isRunning = getLaunches()
-				.filter(l -> !l.isTerminated() && launchMatches(l, launchConfiguration, localDebugPort))
+				.filter(l -> !l.isTerminated() && launchMatches(l, launchConfiguration))
```

Comment:
re-read your comment once again. The reason for not just ignoring it is that Sonar has the rule:  "Unused method parameters should be removed". And i agree with it, because imagine the case when you want to use some method, that has this unused parameter. And it's not easy for you retrieve it at the moment at your place of code to pass it into this method. Of course, you can read the method body and understand that it's not used and can be deleted, but it's unnecessary work during coding + you'd probably write a new private method for your needs. So i think it's better to refactor it while refactoring in this PR